src/i18n/locale/en.json

```diff
@@ -211,6 +212,18 @@
   "components.RequestModal.seasonnumber": "Season {number}",
   "components.RequestModal.selectseason": "Select season(s)",
   "components.RequestModal.status": "Status",
+  "components.ResetPassword.confirmpassword": "Confirm Password",
+  "components.ResetPassword.email": "Email",
+  "components.ResetPassword.emailresetlink": "Email me a Recovery Link",
+  "components.ResetPassword.forgotpassword": "Forgot your password?",
+  "components.ResetPassword.gobacklogin": "Go Back to Sign-In Page",
+  "components.ResetPassword.password": "Password",
+  "components.ResetPassword.resetpassword": "Reset Password",
+  "components.ResetPassword.successmessage": "A password reset link will be sent to the provided email address if it is associated with a valid user.",
```

Comment:
Your i18n strings are overriding each other because the components are in the same folder. Either give them different keys or nest the components in their own folders.